ntotsc/ntotsc.tp2

```diff
@@ -100,7 +100,7 @@ ALWAYS
 				"schinese" => "CP936"
 			END
 			ACTION_DEFINE_ARRAY charsetsReloadArray BEGIN setup journal END
-//			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN ## END
+//			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN setup-osx setup-unix setup-win32 END
```

Comment:
This line is commented out. Have you tested your solution?

Reply:
yes, it need to delete //.